process/block/preprocess/smartContractResults.go

```diff
@@ -333,13 +333,6 @@ func (scr *smartContractResults) SaveTxsToStorage(body *block.Body) error {
 		if miniBlock.Type != block.SmartContractResultBlock {
 			continue
 		}
```

Comment:
It is still little bit useless to these miniblocks. This is mostly needed by destination shard. not sender.